python_bindings/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = [
+  "setuptools>=42",
+  "wheel",
+  "scikit-build",
+  "pybind11==2.6.2",
```

Comment:
orthogonal to this PR, but we should probably document why we use this version of pybind11; IIRC it's because it's a version that we believe to be readily available on multiple distros?

Reply:
ah, nm, it's `scikit-build` in pip

Reply:
>IIRC

I no longer recall, so to-do in another PR. Maybe create an issue about upgrading it? Maybe there were breaking API changes?